pages/admin/request/[id].tsx

```diff
@@ -26,11 +30,22 @@ type Props = {
  */
 const Request: NextPage<Props> = ({ id: idString }: Props) => {
   const id = parseInt(idString);
-
+  const [permitHolderID, setPermitHolderID] = useState<number | undefined>(undefined);
```

Comment:
This state variable isn't needed. After querying for the applicant ID in the existing GQL query in this file, just destructure it from `data`